lib/vsc/filesystem/oceanstor.py

```diff
@@ -1069,6 +1118,29 @@ def set_user_quota(self, soft, user, obj=None, hard=None, inode_soft=None, inode
         @type inode_soft: integer with soft limit on files.
         @type inode_soft: integer with hard limit on files. If None, OCEANSTOR_QUOTA_FACTOR * inode_soft.
         """
+        # TODO: remove (1) and (2) once OceanStor supports setting user quotas on non-empty filesets
+        # (1) Always set default user quotas for all users, instead of quotas specific to each user
+        user = '*'
```

Comment:
can we get a debug messing here about this? Makes it slightly easier to not forgot in the future :wink:

Reply:
let's print a warning, done in https://github.com/vub-hpc/vsc-filesystem-oceanstor/pull/7/commits/0f5e1ace282b88201cfa1e1e4a32616d85363897

Reply:
Warning is going to show up every time this is used...